pkg/pillar/cmd/zedagent/handlentp.go

```diff
@@ -188,7 +194,7 @@ func createNTPSource(s *chrony.ReplySourceData,
 		Poll:    int32(s.Poll),
 		Stratum: uint32(s.Stratum),
 		// Address of the NTP peer, so destination
-		DstAddr: s.IPAddr.String(),
+		DstAddr: dstAddr,
```

Comment:
Personally, I'm not really a fun of reverse lookups. They can introduce delays and return awkward hostnames. Reverse lookup uses PTR records, which are maintained separately from the A/AAAA records (for forward lookups). The PTR records are typically controlled by the ISP or organization that owns the IP address block. For example, while `0.sk.pool.ntp.org` resolves to `212.55.254.189` in my case, reverse lookup of that IP returns `139251.ba.adsl.slovanet.sk.`
For this reason, I always run `iptables -L` and `tcpdump` with `-n`.
I would prefer if we keep IPs as stable NTP source IDs and add hostname as an additional field (which may or may not be filled by EVE). Also, if there is something in the NTP protocol as "NTP source name", I would use that instead (don't know if there is such field).

Reply:
Let's drop this patch for now,